### ISSUE TITLE
Minor notational point

### ISSUE THREAD

**Issue by sethaxen** [mjhajharia/transforms#56]
In the paper, we define the $N$-simplex as being an $N+1$-length vector. In our model implementations, we use $N$ to refer to the number of elements in the vector on the simplex. We should harmonize these notations.

**Comment by mjhajharia** [comment]
yeah agreed, also remembered @Raoul-Kima 's comment on the summation having N which should be replaced to N+1 in the paper. I saw overleaf now, and- that is incorrect, it would remain N, reason being, this is a unit simplex not the usual simplex, the unit N simplex is n-1 dimensional, which means N points. and the usual simplex is n dimensional (n+1 points)

**Comment by mjhajharia** [comment]
regarding this, unit simplex is again incorrect terminology i think, we need to be more clear about this - are we writing things in terms of the identified version of the probability simplex(unit simplex) or just the probability simplex which is n-1 dimensional. for example boyd (popular convex optimization book) defines

<img width="780" alt="image" src="https://user-images.githubusercontent.com/55153708/187544557-5f1855c7-9ded-4db0-a22c-f57c3f1d6e43.png">

**Comment by mjhajharia** [comment]
made a comment and removed it (just me mixing up notation *again*, which makes the need to make it clear even more important). i keep thinking in terms of $\Delta^N$ and forgetting we're actually using $\Delta^{N-1}$ but defining $\Delta^{N}$ in the first section

@Raoul-Kima if you got a github email, ignore that!

**Comment by bob-carpenter** [comment]
We want the standard definition of unit simplex,

$\Delta^N = \\{ \theta \in \mathbb{R}^{N+1} : \theta_n \geq 0, \textrm{sum}(\theta) = 1 \\}.$

Then we just have to keep the dimensions consistent elsewhere.   I'm about to take a notational pass through the doc.

**Comment by bob-carpenter** [comment]
Did everyone decide to just skip over doing the simplex version and go straight to full generality?   The intro no longer mentions simplexes and Section 3 currently begins,

> In this section, we consider transforms from unconstrained space to constrained vectors, including
unit-simplex, sum-to-zero, and unit-length constraints

Given the change in focus, I didn't want to get in an edit war of trying to change it back, so I just left it.

P.S.  I didn't know where to comment with so many issues, so just chose this one at random.

P.P.S.  I like the newly framed transforms intro section.  That's about as clear as I think we'll be able to be.

**Comment by mjhajharia** [comment]
> Did everyone decide to just skip over doing the simplex version and go straight to full generality? The intro no longer mentions simplexes and Section 3 currently begins,
> 
> > In this section, we consider transforms from unconstrained space to constrained vectors, including
> > unit-simplex, sum-to-zero, and unit-length constraints
> 
> Given the change in focus, I didn't want to get in an edit war of trying to change it back, so I just left it.
> 
> P.S. I didn't know where to comment with so many issues, so just chose this one at random.
> 
> P.P.S. I like the newly framed transforms intro section. That's about as clear as I think we'll be able to be.

i thought we agreed on simplex. and not general. i was making rough patches of things about aitchison geometry in general to add, @spinkney adding the ilr as well makes it seem like a good prospect to me - so we get a chance to introduce the transforms idea for hmc in general as well as the compositional data analysis utility point of view is there

yeah i think it looks good enough!

PS - sorry for not responding a lot on the issues, missed 3 weeks of classes so had some catching up to do, will add my changes in a day or two, the cluster had some rotation shift going on, so will check check and then put the other transforms to sample as well.

**Comment by mjhajharia** [comment]
> We want the standard definition of unit simplex,
> 
> ΔN={θ∈RN+1:θn≥0,sum(θ)=1}.
> 
> Then we just have to keep the dimensions consistent elsewhere. I'm about to take a notational pass through the doc.

yeah i continued this over the email thread - unit and standard simplex have similar definitions and some places use them interchangeably which isnt correct and then there's aitchison's definition too

**Comment by bob-carpenter** [comment]
The "unit" in "unit simplex" refers to the sum being unity (i.e., 1).  In the [Wikipedia entry for simplex](https://en.wikipedia.org/wiki/Simplex), "standard N-simplex" and "unit N-simplex" are synonyms that both refer to $\Delta^{N}$.  I suggest we just stick to the "unit simplex" terminology so as not to cause confusion.
